legislature/models.py

```diff
@@ -94,7 +64,44 @@ def votes_with_proposal(self):
         ).filter(representative=self)
 
 
-@receiver(post_save, sender=Representative)
-def create_memopolrepresentative_from_representative(instance, **kwargs):
-    memopol_representative = create_child_instance_from_parent(MemopolRepresentative, instance)
-    memopol_representative.save()
+def mempol_representative(sender, representative, data, **kwargs):
```

Comment:
The name of this function doesn't mean anything at all, this makes code hard to maintain.